components/taupage.rst

```diff
@@ -542,20 +543,64 @@ And the Agent will follow these logs:
 
 You can get your Account Key from the Logentries Webinterface under /Account/Profile
 
-
 scalyr_account_key
 ------------------
 
 **(optional)**
 
+Options: see :ref:`scalyr_account_key <scalyr_account_key>` in :ref:`logging <logging>`
+
+scalyr_application_log_parser
+-----------------------------
+
+**(optional)**
+
+Options: see :ref:`scalyr_application_log_parser <scalyr_application_log_parser>` in :ref:`logging <logging>`
+
+scalyr_custom_log_parser
+------------------------
+
+**(optional)**
+
+Options: see :ref:`scalyr_custom_log_parser <scalyr_custom_log_parser>` in :ref:`logging <logging>`
+
+scalyr_region
+-------------
+
+**(optional)**
+
+Options: see :ref:`scalyr_region <scalyr_region>` in :ref:`logging <logging>`
+
+.. _logging:
+
+logging
+-------
+
+Sample logging configuration::
+
+     logging:
+         fluentd_enabled: true
+         fluentd_loglevel: info
+         s3_bucket: log-bucket-eu-central-1
+         log_destination: s3
+         authlog_destination: scalyr
+         scalyr_region: eu
+         scalyr_account_key: "aws:kms:XYZABC..."
+
+In this example everything but the ``auth.log`` is logged to ``s3``, the ``auth.log`` is logged to ``Scalyr``. Logging is done with ``Fluentd``.
+
 .. NOTE::
-   You can also use AWS KMS to encrypt your Scalyr account key. See in the example above.
+   If a logging section is present in ``senza.yaml`` then ``scalyr_account_key``, ``scalyr_application_log_parser``, ``scalyr_custom_log_parser`` and ``scalyr_region`` entries outside the scope of the logging section will be ignored.
 
-If you provide the Scalyr AccountKey in the .yaml file, the agent of Scaylr will be installed and will follow these logs:
+.. _scalyr_account_key:
 
-  * /var/log/syslog
-  * /var/log/auth.log
-  * /var/log/application.log
```

Comment:
Is this removed from the documentation (which logs are sent by default)? I don't see it being moved somewhere else.

Reply:
added back to logging -> log_destination